airflow/update-environment-variables.sh

```diff
@@ -1,31 +1,40 @@
 #!/usr/bin/env bash
 echo Input ingestion service url?
-read ingestionUrl
+INGESTION_URL=$(gcloud run services list --platform managed --filter ingestion-service --format 'value(status.url)')
+echo "Using Fetched Data: ${INGESTION_URL}"
 
 echo Input dedupe service url?
-read dedupeUrl
+DEDUPE_URL=$(gcloud run services list --platform managed --filter dedupe-service --format 'value(status.url)')
+echo "Using Fetched Data: ${DEDUPE_URL}"
 
 echo Input gcs to bq service url?
-read gcsToBqServiceUrl
+GCS_TO_BQ_URL=$(gcloud run services list --platform managed --filter gcs-to-bq-service --format 'value(status.url)')
+echo "Using Fetched Data: ${GCS_TO_BQ_URL}"
 
 echo Input exporter service url?
-read exporterServiceUrl
+EXPORTER_URL=$(gcloud run services list --platform managed --filter exporter-service --format 'value(status.url)')
+echo "Using Fetched Data: ${EXPORTER_URL}"
 
 echo Input aggregator service url?
-read aggegatorServiceUrl
+AGGREGATOR_URL=$(gcloud run services list --platform managed --filter aggregator-service --format 'value(status.url)')
+echo "Using Fetched Data: ${AGGREGATOR_URL}"
 
 echo Input landing bucket?
-read landingBucket
+LANDING_BUCKET=$(gsutil ls | grep -oP ".*landing.*")
+LANDING_BUCKET=${LANDING_BUCKET:5:-1}
+echo $LANDING_BUCKET
```

Comment:
Could you echo a more descriptive message for LANDING_BUCKET and MANUAL_BUCKET, eg "Using ${LANDING_BUCKET} as landing bucket" and similar for manual bucket?